src/docfx/restore/lock/DependencyLock.cs

```diff
@@ -10,21 +10,21 @@ namespace Microsoft.Docs.Build
 {
     internal static class DependencyLock
     {
-        public static DependencyLockModel GetGitLock(this DependencyLockModel dependencyLock, string href, string branch)
+        public static (string key, DependencyLockModel value) GetGitLock(this DependencyLockModel dependencyLock, string href, string branch)
```

Comment:
consider not include this change in this PR.

Reply:
i will try to not change the structure